content/doc/addons/mongodb/_index.md

```diff
@@ -78,4 +78,20 @@ Note that these features are available for all our databases add-ons, in additio
 
 **The managed databases provide an advantage if you want to avoid the complexity of installing and deploying databases** and if you want simple-to-use management and monitoring tools.
 
-{{% content/managed-services %}}
+## 🔑 Rights and permissions
+
+Add-ons are managed services, meaning that users have **standard access** to the database (**owner** of the database). Some operations like database and user creation, as well as some settings modifications aren't available by default. This ensures optimal performances and security for managed services as configured by Clever Cloud.
```

Comment:
I think we should try to avoid repetitions of words such as `database` as much as we can (not simple on such parts of the documentation, but it makes it better to read)

Reply:
Ok I removed the repetition